### DISCUSSION TITLE
How to improve efficiency?

### DISCUSSION THREAD

**Discussion by flohorovicic** [cgre-aachen/bayseg#29] [General]
How can we improve the efficiency?

**Comment by hwang051785** [comment]
I think wrapping up the sampling steps in C++ with OpenMP could be a solution.

**Reply by flohorovicic** [reply]
Hi Jack,

Yes, that could indeed be a solution! I am meeting with Christoph on Wednesday and we will have a look at a potential implementation into a ML library, as a first step (or JAX). Let's see how that goes... 

Greetings form Aachen!
